litmus-portal/frontend/src/components/WelcomeModal/Stepper.tsx

```diff
@@ -61,18 +70,24 @@ function CStepper() {
         if ('error' in data) {
           setFormError(true);
         } else {
-          user.updateUserDetails({
-            name: data.name,
-            email: data.email,
+          CreateUser({
+            variables: {
+              user: {
+                username: userData.username,
+                email: info.email,
+                name: info.name,
+                project_name: info.projectName,
```

Comment:
These should be `data` and not `info` since we are doing a `.then(data)` after the response

Reply:
It was `data` earlier but not getting updated in the DB earlier I guess, setting it to `info` so that whatever user updates in the modal, we send it for mutation. cc: @Sanjay1611